src/reference/controlflow.md

```diff
@@ -104,7 +104,7 @@ The compiler then transforms the statement into byte code equivalent to the
 statement rewritten to
 
 ```
-🍡 iterable❗️ ➡️ iterator
+🍡 iteratable❗️ ➡️ iterator
```

Comment:
The word "iterable" is commonly used. See for example [Java](https://docs.oracle.com/javase/8/docs/api/java/lang/Iterable.html) or [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols). I don't think have read "iteratable" before.